ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,225 @@
+
+            Select(exOne, SortedBy.Asc);
+            Print(exOne);
+
+            Console.WriteLine();
+
+            Select(exOne, SortedBy.Desc);
+            Print(exOne);
+
+            Console.WriteLine();
+
+            Console.WriteLine("Bubble sort: ");
+            Bubble(exOne, SortedBy.Asc);
+            Print(exOne);
+
+            Console.WriteLine();
+
+            Bubble(exOne, SortedBy.Desc);
+            Print(exOne);
+
+            Console.WriteLine();
+
+            Console.WriteLine("Insert sort: ");
+
+            Insert(exOne, SortedBy.Asc);
+            Print(exOne);
+
+            Console.WriteLine();
+
+            Insert(exOne, SortedBy.Desc);
+            Print(exOne);
+
+            Console.WriteLine();
+
+            Console.WriteLine("Enums: ");
+
+
+
+            Sort(exTwo, SortAlgorithmType.Select, SortedBy.Asc);
+            Sort(exTwo, SortAlgorithmType.Select, SortedBy.Desc);
+            Console.WriteLine();
+
+            Sort(exTwo, SortAlgorithmType.Bubble, SortedBy.Asc);
+            Sort(exTwo, SortAlgorithmType.Bubble, SortedBy.Desc);
+            Console.WriteLine();
+
+            Sort(exTwo, SortAlgorithmType.Insert, SortedBy.Asc);
+            Sort(exTwo, SortAlgorithmType.Insert, SortedBy.Desc);
+            Console.WriteLine();
+        }
+        static void Select(int[] ints, SortedBy sortedBy)
+        {
+            int x = ints.Length;
+            switch (sortedBy)
+            {
+                case SortedBy.Asc:
+
+                    for (int i = 0; i < x - 1; i++)
+                    {
+                        int min = i;
+                        for (int j = i + 1; j < x; j++)
+                        {
+                            if (ints[j] < ints[min])
+                                min = j;
+                        }
+                        int temp = ints[min];
+                        ints[min] = ints[i];
+                        ints[i] = temp;
+                    }
+
+                    break;
+                case SortedBy.Desc:
+                    for (int i = 0; i < x - 1; i++)
+                    {
+                        int min = i;
+                        for (int j = i + 1; j < x; j++)
+                        {
+                            if (ints[j] > ints[min])
+                                min = j;
+                        }
+                        int temp = ints[min];
+                        ints[min] = ints[i];
+                        ints[i] = temp;
+
+                    }
+                    break;
+            }
+        }
+        static void Print(int[] ints)
+        {
+            for (int i = 0; i < ints.Length; i++)
+            {
+                Console.Write(ints[i] + " ");
+            }
+        }
+        static void Bubble(int[] ints, SortedBy sortedBy)
+        {
+            int x = ints.Length - 1;
+            switch (sortedBy)
+            {
+                case SortedBy.Asc:
```

Comment:
You have almost identical 2 method's pieces, which differ only in comparing elements
So you need to
sortedBy == SortedBy.Asc ? ints[j] > ints[j + 1] : ints[j] < ints[j + 1]